tests/test_config.py

```diff
@@ -522,6 +544,8 @@ def _mock_isfile(filename):
 async def test_loading_configuration_from_packages(hass):
     """Test loading packages config onto hass object config."""
     hass.config = mock.Mock()
+    hass.config.load.return_value = asyncio.Future()
```

Comment:
you should be able to drop this. Storage is mocked out in all function based tests.

Reply:
Yes, fixed.